packages/strapi-plugin-preview/extensions/content-manager/admin/src/containers/ListView/index.js

```diff
@@ -9,28 +9,29 @@ import React, {
 import PropTypes from "prop-types";
```

Comment:
diff showing changes to original strapi code are visible in fork:
https://github.com/VirtusLab/strapi/pull/2/files